boltkit/controller.py

```diff
@@ -398,12 +393,12 @@ class Cluster:
     def __init__(self, path):
         self.path = path
 
-    def install(self, version, core_count, read_replica_count, password, verbose=False):
+    def install(self, version, core_count, read_replica_count, initial_port, password, verbose=False):
```

Comment:
This adds a positional argument mid-signature. Will this break any existing calls?